Documentation/ceph-cluster-crd.md

```diff
@@ -67,6 +67,8 @@ This feature is only available when `useAllNodes` has been set to `false`.
 - `count`: set the number of mons to be started. The number should be odd and between `1` and `9`. If not specified the default is set to `3` and `allowMultiplePerNode` is also set to `true`.
 - `allowMultiplePerNode`: enable (`true`) or disable (`false`) the placement of multiple mons on one node. Default is `false`.
 
+If these settings are changed in the CRD the operator will update the number of mons during a periodic check of the mon health, which by default is every 45 seconds.
```

Comment:
How about noting how the mon out timeout option is named and/or pointing to a doc page where this documented?